### ISSUE TITLE
Empty mnemonic display

### ISSUE THREAD

**Issue by despens** [nextcloud/desktop#2327]
### Expected behaviour
Mnemonic should be displayed.

### Actual behaviour
Mnemonic field is empty.

### Steps to reproduce
1. Click on "Display mnemonic" button.

### Client configuration
Client version: 3.0.0 (Ubuntu)
Operating system: Ubuntu 20.04
OS language: US English
Qt version used by client package (Linux only, see also Settings dialog): n/a, installed via official deb repository
Client package (From Nextcloud or distro) (Linux only): nextcloud-desktop 3.0.0-20200821.171045.ff175088a-1.0~focal1 amd64
Installation path of client: /usr/bin/nextcloud

### Server configuration
Nextcloud version: 19.0.1

Storage backend (external storage): Local file storage (default), s3 via minio

![image](https://user-images.githubusercontent.com/571494/91410273-f55ba480-e846-11ea-9672-475aaae8f06d.png)

**Comment by camilasan** [comment]
Make sure that before you open an bug report:
- in the case of end to end encryption related problems, the desktop client must be at least 3.0 and the server at least 19 with the end to end encryption server app version at least 1.5.2. Check [https://apps.nextcloud.com/apps/end_to_end_encryption](https://apps.nextcloud.com/apps/end_to_end_encryption).
- If you ever tried end-to-end encryption before version 3.0, follow the instructions from this post to clean up the keys as that can break the functioning of >3.0 if you had a malformed key:
[https://help.nextcloud.com/t/help-test-the-latest-version-of-e2ee/87590](https://help.nextcloud.com/t/help-test-the-latest-version-of-e2ee/87590).
- if that doesn't work we need the logs since we don't know how to do magic here :) - run the client with --logdebug. See instructions in the documentation: [https://docs.nextcloud.com/desktop/2.6/troubleshooting.html#log-files](https://docs.nextcloud.com/desktop/2.6/troubleshooting.html#log-files).

Re-open the issue once you have logs. Thanks!

**Comment by despens** [comment]
It looks like the issue is that end-to-end encryption on my nextcloud v 19.0.1 is not installed, but the client tells me it is supported:

![image](https://user-images.githubusercontent.com/571494/91414791-5ab29400-e84d-11ea-8513-bd1c54c95518.png)

Thank you for pointing me towards the logging functionality of the client!

This looks like the relevant part, in which the client recognizes no e2e is available, but still draws the button:

```
2020-08-27 10:15:02:301 [ info nextcloud.sync.credentials.webflow ]:	request finished
2020-08-27 10:15:02:301 [ info nextcloud.sync.clientsideencryption ]:	Initializing
2020-08-27 10:15:02:301 [ info nextcloud.sync.clientsideencryption ]:	No Client side encryption available on server.
2020-08-27 10:15:02:302 [ info nextcloud.gui.account.state ]:	AccountState connection status change:  OCC::ConnectionValidator::Undefined -> OCC::ConnectionValidator::Connected
2020-08-27 10:15:02:302 [ info nextcloud.gui.account.state ]:	AccountState state change:  "Disconnected" -> "Connected"
2020-08-27 10:15:02:302 [ info nextcloud.gui.activity ]:	Start fetching activities for  "▮▮▮▮▮▮▮▮▮@▮▮▮▮▮▮▮▮"
2020-08-27 10:15:02:302 [ info nextcloud.sync.accessmanager ]:	2 "" "https://▮▮▮▮▮▮▮▮/ocs/v2.php/apps/activity/api/v2/activity?since=0&limit=50&format=json" has X-Request-ID "b273807e-2e35-4723-a96f-b2a3195b6a1d"
2020-08-27 10:15:02:302 [ info nextcloud.sync.networkjob ]:	OCC::JsonApiJob created for "https://▮▮▮▮▮▮▮▮" + "ocs/v2.php/apps/activity/api/v2/activity" "OCC::ActivityListModel"
2020-08-27 10:15:02:302 [ info nextcloud.sync.accessmanager ]:	2 "" "https://▮▮▮▮▮▮▮▮/ocs/v2.php/apps/notifications/api/v2/notifications?format=json" has X-Request-ID "bc9e99fe-26fa-491d-96e8-b4d92043f12b"
2020-08-27 10:15:02:302 [ info nextcloud.sync.networkjob ]:	OCC::JsonApiJob created for "https://▮▮▮▮▮▮▮▮" + "ocs/v2.php/apps/notifications/api/v2/notifications" "OCC::ServerNotificationHandler"
2020-08-27 10:15:02:303 [ info nextcloud.gui.account.settings ]:	Account "▮▮▮▮▮▮▮▮@▮▮▮▮▮▮▮▮" Client Side Encryption false
```

Let me know if this is enough information for you to continue.

**Comment by camilasan** [comment]
> the client recognizes no e2e is available, but still draws the button:

The good news is that this is fixed and today I will be tagging 3.0.1 with the fix for it (PR #2280)

Anyhow thanks for the logs :)
